accepted/future-releases/extension-types/feature-specification.md

```diff
@@ -1158,7 +1163,7 @@ representation type `R`, and that the extension type `V1` with
 declaration _DV1_ is a superinterface of _DV_ (*note that `V1` may
 have some actual type arguments*).  Assume that `S` is the
 instantiated representation type corresponding to `V1`. A compile-time
-error occurs if `R` is not a subtype of `S`.
+error occurs if `R` is not a subtype of `S` nor a subtype of `V1`.
```

Comment:
Nit: use "neither" to match "nor".
Otherwise use "and not".

Reply:
Of course, done!